main.go

```diff
@@ -127,9 +127,61 @@ func messageCreate(s *discordgo.Session, m *discordgo.MessageCreate) {
 		return
 	}
 
-	// Check if it's a command we recognize
+	if strings.HasPrefix(m.Content, "!skill") {
+		//TODO: Finish this command
+		//cmdSkill(m.Content, channel, s, m)
+	}
+
 	if strings.HasPrefix(m.Content, "!leaguewatch") {
-		tokens := strings.Split(m.Content, " ")
+		cmdLeagueWatch(m.Content, channel, s, m)
+	}
+
+	if strings.HasPrefix(m.Content, "!help") {
+		cmdHelp(channel, s, m)
+	}
+
+
+}
+
+func cmdSkill(c string, c2 *discordgo.Channel, s *discordgo.Session, m *discordgo.MessageCreate) {
```

Comment:
`c` and `c2` are bad variable names, (So are `s` and `m` but we can fix those later)